screens/index.html

```diff
@@ -0,0 +1,410 @@
+          </div>
+        </div>
+      </div>
+    </header>
+    <div class="container">
+      <div class="section1 m-10">
+        <div class="wrapSection1 row">
+          <h2>
+            <span>Lorem</span>
+            <span>ipsum</span>   
+          </h2>
+          <p class="wrapSection1-descripstion  c-10 c-o-1">
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt. Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora incidunt ut labore et dolore magnam aliquam quaerat voluptatem. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit
+          </p>
+        </div>
+      </div>
+
+      <div class="section3">
+        <div class="wraps3 grid wide ">
+          <div class="wrapsec3 row m-10 c-10 ">
+            <div class="colsec3 col l-4-ex m-5 c-12">
+              <div class="wraper-col">
+                <div class="col-sec3-title">
+                  <h2 class="uppercase">
+                    beginner
+                  </h2>
+                  <div class="col-sec3-price">
+                    <span>$99</span>
+                    <span>Per month</span>
+                  </div>
+                  <p class="displaynone">The best price,quality plan</p>
+                </div>
+                <div class="col-sec3-sub">
+                  <h2 class="">
+                    Subtiitle goes here
+                  </h2>
+                  <p>
+                    Subtitle second line goes here lorem ipsum
+                  </p>
+                </div>
+                <ul class="con-sec3-list">
+                  <li class="con-sec3-list__color">Drop out  reman husle</li>
+                  <li>Drop out reman husle sdasa sdJJ</li>
+                  <li class="con-sec3-list__color">Drop out  reman husle  fsfa</li>
+                  <li class="line-thought"> Drop out  reman husleffffffffffffz</li>
+                  <li class="con-sec3-list__color">Drop out  reman husle</li>
+                </ul>
+                
+                <div class="col-sec3-btn">
+                  <button class=""><span class="triangle-sec3"></span>Order</button>
```

Comment:
em dùng icon của font awesome em nhé, tương tự cho các chỗ khác
```suggestion
                  <button class=""><i class="fa fa-caret-down rotate"></i>Order</button>
```